repository.go

```diff
@@ -150,6 +150,14 @@ func (r *repository) getToggle(key string) *api.Feature {
 	return nil
 }
 
+func (r *repository) list() []api.Feature {
+	var features []api.Feature
```

Comment:
You should probably lock the mutex for reading here before reading from the store. This should just be a matter of adding:

```go
r.RLock()
defer r.RUnlock()
```
to the top of the function.